babel.config.js

```diff
@@ -10,6 +10,5 @@ module.exports = {
       }
     ],
     '@babel/preset-typescript'
-  ],
-  plugins: ['@babel/proposal-class-properties']
```

Comment:
From the NPM page: https://www.npmjs.com/package/@babel/plugin-proposal-class-properties

> This proposal has been merged to the ECMAScript standard and thus this plugin is no longer maintained. Please use @babel/plugin-transform-class-properties instead.

Upon inspection of the new site ( https://babeljs.io/docs/babel-plugin-transform-class-properties ). It appears this plugin is safe to drop because it's been moved to `@babel/preset-en` as long as you support ES2022 .